components/main_page/wikis/starcraft2/main_page_layout_data.lua

```diff
@@ -0,0 +1,189 @@
+			'<div style{{=}}"display:inline; float:right;" class="plainlinks smalledit">' ..
+			'&#91;[[Special:EditPage/Player Transfers/{{Current term}}|edit]]&#93;</div>\n' ..
+			'<div style{{=}}"white-space:nowrap; display:inline; margin:0 10px font-size:15px; font-style:italic;">' ..
+			'[[Transfers|See more transfers]]<span style="font-style:normal; padding:0 5px;">&#8226;</span>' ..
+			'[[lpcommons:Special:RunQuery/Transfer|Input Form]]</div>\n</div>',
+		boxid = 1509,
+	},
+	thisDay = {
+		heading = 'This day in StarCraft II <small id="this-day-date" style = "margin-left: 5px">' ..
+			'({{#time:F}} {{Ordinal|{{#time:j}}}})</small>',
+		body = '{{Liquipedia:This day}}',
+		padding = true,
+		boxid = 1510,
+	},
+	specialEvents = {
+		noPanel = true,
+		body = '{{Liquipedia:Special Event}}',
+		boxid = 1511,
+	},
+	filterButtons = {
+		noPanel = true,
+		body = '<div style{{=}}"width:100%;margin-bottom:8px;">' ..
+			'{{#invoke:Lua|invoke|module=FilterButtons|fn=getFromConfig}}</div>',
+	},
+	matches = {
+		heading = 'Matches',
+		body = '{{#invoke:Lua|invoke|module=Widget/Factory|fn=fromTemplate|widget=Match/Ticker/Container}}' ..
+			'<div style{{=}}"white-space:nowrap; display: block; margin:0 10px; ' ..
+			'font-size:15px; font-style:italic; text-align:center;">[[Liquipedia:Matches|See more matches]]</div>',
+		padding = true,
+		boxid = 1507,
+	},
+	tournaments = {
+		heading = 'Tournaments',
+		body = '{{#invoke:Lua|invoke|module=Widget/Factory|fn=fromTemplate|widget=Tournaments/Ticker' ..
+			'|upcomingDays=7|completedDays=7}}',
+		padding = true,
+		boxid = 1508,
+	},
+}
+
+return {
+	banner = {
+		lightmode = 'Starcraft-2-logo-lightmode.svg',
+		darkmode = 'Starcraft-2-logo-darkmode.svg',
+	},
+	metadesc = 'Comprehensive StarCraft II wiki with articles covering everything from units and buildings, ' ..
+		'to strategies, to tournaments, to competitive players and teams.',
+	title = 'StarCraft II',
+	navigation = {
+		{
+			file = 'Kelazhur IEM Shenzhen 2015.jpg',
+			title = 'Players',
+			link = 'Portal:Players',
+			count = {
+				method = 'LPDB',
+				table = 'player',
+			},
+		},
+		{
+			file = 'Kelazhur IEM Shenzhen 2015.jpg',
+			title = 'Teams',
+			link = 'Portal:Teams',
+			count = {
+				method = 'LPDB',
+				table = 'team',
+			},
+		},
+		{
+			file = 'Kelazhur IEM Shenzhen 2015.jpg',
+			title = 'Tournaments',
+			link = 'Portal:Tournaments',
+			count = {
+				method = 'LPDB',
+				table = 'tournament',
+			},
+		},
+		{
+			file = 'Kelazhur IEM Shenzhen 2015.jpg',
+			title = 'Transfers',
+			link = 'Portal:Player transfers',
+			count = {
+				method = 'LPDB',
+				table = 'transfer',
+			},
+		},
+		{
+			file = 'Kelazhur IEM Shenzhen 2015.jpg',
+			title = 'Statistics',
+			link = 'Portal:Statistics',
+		},
+		{
+			file = 'Kelazhur IEM Shenzhen 2015.jpg',
+			title = 'ESL Pro Tour',
+			link = 'Portal:2023 EPT',
+		},
```

Comment:
likely to be removed end of the month and replaced with something else
(i think if we do not get an announcement until then we can assume sc2 has no circuit anymore :( )